### ISSUE TITLE
OCR service

### ISSUE THREAD

**Issue by traffic1000** [geoadmin/web-asset-swissgeol#17]
Issue taken from previous Taks list (#6)

Backend OCR service, same as in digi.swissgeol.ch

1. Automatically queue all not OCR processed documents and send job to OCR service
2. Save textbody in asset['textbody'] --> tbd if this is necessary? / OCR layer within document is enough
3. Update StatusWork when OCR is successfully finished: status_work['status_work_item'] == 'OCRprocessed' (from value list) & status_work['status_work_date'] == timestamp",

**Comment by daniel-va** [comment]
@stijnvermeeren-swisstopo Soweit wir sehen ist das obsolet / noch Teil des alten Algorithmus. Können wir das schliessen?

**Comment by stijnvermeeren-swisstopo** [comment]
@daniel-va Ja, für mich können wir dieses Issue schliessen. Objekterkennung in Assets ist im Moment keine Priorität. Zwar wird das Thema sicher mittelfristig wieder aktuell werden, aber wahrscheinlich sind die Anforderungen dann sowieso ziemlich anders als in diesem altern Issue, und macht es mehr Sinn dann ein neues Ticket zu erstellen,
